web-browsers/browsers-and-dev-tools.md

```diff
@@ -0,0 +1,109 @@
+# Browsers and Dev Tools
+
+## Vocabulary and Synonyms
+
+| Vocab     | Definition                              | Synonyms | How to Use in a Sentence                                                                                                                   |
+| --------- | --------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| Rendering | Displaying something visual to the user | Showing  | "How the Firefox browser renders that web page may be different compared to how Google Chrome renders it. That's why they look different." |
+
+## The Browser Platform
+
+The _browser_ is the platform that runs web apps in front of an end user.
+
+When we develop the front-end layer for our web apps in code, the _browser_ is the platform that _renders_ it for the user. **Rendering** is the process of displaying it to the user.
+
+As web developers, familiarity with the browser will help us understand its abilities and limitations. Understanding what the browser can and can't do will help us make usable, exciting, consistent web apps!
+
+## Developing for Web
+
+As web developers, we will write code for our browser to render or run.
+
+There are three separate responsibilities to consider when making a website:
+
+1. Content
+1. Presentation
+1. Behavior
+
+### Content
+
+The _content_ of a website is all of its text, paragraphs, links, sections, headers, images, videos, etc.
+
+Imagine a recipe blog. The contents of a recipe blog include:
+
+- The paragraphs of story before the recipe
+- The list of ingredients
+- The directions and steps to talk
+- All images and photos related to the recipe
+- Navigational links
+- The logo of the recipe website and a link to go home
+- The footer content that lists contact information about the author
+
+### Presentation
+
+The _presentation_ of a website is how the content looks on the page. This includes colors, fonts, layout, background images, and more.
+
+In our fictional recipe blog, the presentation of the blog determines:
+
+- The typeface, font size, and text color of the recipe
+- If the list of ingredients is displayed vertically or horizontally
+- The border and sizing of all images and photos
+- If the navigational links are on the side of the website, on top, or on bottom
+- If the logo changes colors when the cursor moves over it
+
+#### Browser Default Styles
+
+It's part of our job to decide how content is presented.
+
+However, all browsers have _default styles_ which will affect our web app presentation. Over time, we'll become familiar with which styles are browser defaults.
```

Comment:
probably super pedantic, but a style reminds me of a specific, singular "theme", whereas styling is a series of rules. Like I said, super pedantic, but that's the feeling this evokes in me.
```suggestion
However, all browsers have _default styling_ which will affect our web app presentation. Over time, we'll become familiar with which styling comes from browser defaults.
```